data/human/human missions.txt

```diff
@@ -3601,3 +3601,76 @@ mission "Mafia Extortion"
 			label payup
 			`	The man takes your credits, responds with a brief "Thank you," and walks away.`
 				accept
+
+
+
+mission "Repulsor failure"
+	landing
+	#Note: this tag applies to the destination, not to the source...
+	infiltrating
```

Comment:
I don't quite follow, why would you have infiltrating, if I follow correctly, this is when you launch in this mission to then land again, the dialog even says the repulsors should work fine on next landing which they do, right? 
if they fail again for some reason, you could add a dialog under `on complete` that explains it

Reply:
I actually wanted to use infiltrating on the initial landing, when the player chooses to take the risk of landing immediately. If you have an emergency landing like that, then the chance of landing near the spaceport is actually quite small.

But I already updated the text about how strange it is that the player lands close to the spaceport.